src/metabase/lib/field.cljc

```diff
@@ -77,6 +79,30 @@
     (when (every? some? path)
       (str/join ": " path))))
 
+(defn- clauses-in-stage-by-uuid
+  [query stage-number]
+  (lib.computed/with-cache-none* query [::clauses-in-stage-by-uuid stage-number]
```

Comment:
Maybe explain on this example how this is useful.

Reply:
It's not, that was leftover from some testing and I forgot about it :( 

I'll put it back to `with-cache-ephemeral*`, assuming the tests actually pass with it.